playground/bicep/BicepSample.AppHost/Program.cs

```diff
@@ -0,0 +1,65 @@
+using Aspire.Hosting.Azure;
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddAzureProvisioning();
+
+var parameter = builder.AddParameter("val");
+
+var templ = builder.AddBicepTemplate("test", "test.bicep")
+                   .WithParameter("test", parameter)
+                   .WithParameter("values", ["one", "two"]);
+
+var kv = builder.AddBicepKeyVault("kv");
+var appConfig = builder.AddBicepAppConfiguration("appConfig");
+var storage = builder.AddAzureBicepStorage("storage");
+                    // .UseEmulator();
+
+var blobs = storage.AddBlob("blob");
+var tables = storage.AddTable("table");
+var queues = storage.AddQueue("queue");
+
+var sqlServer = builder.AddBicepAzureSql("sql").AddDatabase("db");
+
+var user = builder.AddParameter("username");
+var pwd = builder.AddParameter("password", secret: true);
+
+var pg = builder.AddBicepAzurePostgres("postgres2", user, pwd).AddDatabase("db2");
```

Comment:
I think for something like this I would be inclined to let the AddBicepAzurePostgres do the wireup to automatically generate the password (using the `"input":` field mechanism).

I think using a parameter is probably more if you want to use an extenral connection string (in this particular case).